src/pyedb/dotnet/edb_core/utilities/simulation_setup.py

```diff
@@ -56,9 +56,12 @@ def __init__(self, pedb, edb_setup=None):
             "kDDRwizard": None,
             "kQ3D": None,
             "kNumSetupTypes": None,
-            "kRaptorX": self._pedb.simsetupdata.RaptorX.RaptorXSimulationSettings,
             "kHFSSPI": self._pedb.simsetupdata.HFSSPISimulationSettings,
         }
+        if self._pedb.edbversion:
+            version = self._pedb.edbversion.split(".")
+            if int(version[0]) == 2024 and int(version[1]) == 2 or int(version[0]) > 2024:
+                self._setup_type_mapping.update({"kRaptorX": self._pedb.simsetupdata.RaptorX.RaptorXSimulationSettings})
```

Comment:
```suggestion
        }
        version = self._pedb.edbversion.split(".")
        if int(version[0]) == 2024 and int(version[1]) == 2 or int(version[0]) > 2024:
            self._setup_type_mapping.update({
                "kRaptorX": self._pedb.simsetupdata.RaptorX.RaptorXSimulationSettings,
                "kHFSSPI": self._pedb.simsetupdata.HFSSPISimulationSettings,
            })
```
As far as I know, kHFSSPI is also not allowed in 2023.2
Also, I removed the check on edbversion. Is there a reason for your if statement ?